docs/tutorials/behavioral_cloning.py

```diff
@@ -0,0 +1,185 @@
+
+
+def collate_fn(batch):
+    return {
+        "id": torch.Tensor([x.id for x in batch]),
+        "seed": torch.Tensor([x.seed for x in batch]),
+        "total_timesteps": torch.Tensor([x.total_timesteps for x in batch]),
+        "observations": torch.nn.utils.rnn.pad_sequence(
+            [torch.as_tensor(x.observations) for x in batch],
+            batch_first=True
+        ),
+        "actions": torch.nn.utils.rnn.pad_sequence(
+            [torch.as_tensor(x.actions) for x in batch],
+            batch_first=True
+        ),
+        "rewards": torch.nn.utils.rnn.pad_sequence(
+            [torch.as_tensor(x.rewards) for x in batch],
+            batch_first=True
+        ),
+        "terminations": torch.nn.utils.rnn.pad_sequence(
+            [torch.as_tensor(x.terminations) for x in batch],
+            batch_first=True
+        ),
+        "truncations": torch.nn.utils.rnn.pad_sequence(
+            [torch.as_tensor(x.truncations) for x in batch],
+            batch_first=True
+        )
+    }
+
+# %%
+# We can now proceed to load the data and create the training loop.
+# To begin, let's initialize the DataLoader, neural network, optimizer, and loss.
+
+
+minari_dataset = minari.load_dataset("CartPole-v1-expert")
+dataloader = DataLoader(minari_dataset, batch_size=256, shuffle=True, collate_fn=collate_fn)
+
+env = minari_dataset.recover_environment()
+observation_space = env.observation_space
+action_space = env.action_space
+assert isinstance(observation_space, spaces.Box)
+assert isinstance(action_space, spaces.Discrete)
+
+policy_net = PolicyNetwork(np.prod(observation_space.shape), action_space.n)
+optimizer = torch.optim.Adam(policy_net.parameters())
+loss_fn = nn.CrossEntropyLoss()
+
+# %%
+# We use the cross-entropy loss like a classic classification task, as the action space is discrete.
+# We then train the policy to predict the actions:
+
+num_epochs = 32
+
+for epoch in range(num_epochs):
+    for batch in dataloader:
+        a_pred = policy_net(batch['observations'][:, :-1])
+        a_hat = F.one_hot(batch["actions"]).type(torch.float32)
+        loss = loss_fn(a_pred, a_hat)
+
+        optimizer.zero_grad()
+        loss.backward()
+        optimizer.step()
+
+    print(f"Epoch: {epoch}/{num_epochs}, Loss: {loss.item()}")
+
+# %%
+# And now, we can evaluate if the policy learned from the expert!
+
+env = gym.make("CartPole-v1", render_mode="human")
+obs, _ = env.reset(seed=42)
+done = False
+accumulated_rew = 0
+while not done:
+    action = policy_net(torch.Tensor(obs)).argmax()
+    obs, rew, ter, tru, _ = env.step(action.numpy())
+    done = ter or tru
+    accumulated_rew += rew
+
+env.close()
+print("Accumulated rew: ", accumulated_rew)
+
+# %%
+# We can visually observe that the learned policy aces this simple control task, and we get the maximum reward.
```

Comment:
this should say what a good reward is for the environment, I got 500 as the accumulated reward when I ran this(Which I assume is a good reward).

Reply:
500 is the maximum obtainable, since after the episode is truncated. I add the explanation